Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -116,11 +116,16 @@ SteamScript.prototype.go = function () {
         .wizard(setupWizard)
         .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion)
         .luna();
+        .corefonts();
```

Comment:
I am nor sure this will works. Also, you did not include the include for corefonts

Reply:
Works for luna verb -> Should work for all verbs. 

corefonts include added.

Reply:
Verify it.

Reply:
My logic was wrong dammit xD fixed